llm_on_ray/inference/predictor_deployment.py

```diff
@@ -394,8 +394,13 @@ async def __call__(self, http_request: Request) -> Union[StreamingResponse, JSON
         streaming_response = request["stream"]
         input = request["text"]
         config = request["config"]
-
+        if config.get("debug_mode", False):
+            print("DEBUG:predictor_deployment.py:print config received from json:", config)
```

Comment:
we already defined a logger in this module (logger = get_logger(__name__)) , could you use it instead of print as print is for normal output.

Reply:
done